packages/beacon-node/src/api/impl/beacon/blocks/utils.ts

```diff
@@ -65,6 +65,14 @@ async function resolveBlockIdOrNull(
     };
   }
 
+  if (blockId === "justified") {
+    const justified = forkChoice.getJustifiedBlock();
+    return {
+      block: await db.block.get(fromHexString(justified.blockRoot)),
+      executionOptimistic: isOptimisticBlock(justified),
```

Comment:
Hey @dapplion, is this what you meant regarding https://github.com/ChainSafe/lodestar/pull/5459#discussion_r1185593338?